WordPress/Classes/Models/DiffAbstractValue+Attributes.swift

```diff
@@ -0,0 +1,53 @@
+extension DiffAbstractValue {
+    var attributes: [NSAttributedString.Key: Any]? {
+        switch operation {
+        case .add:
+            return [.backgroundColor: WPStyleGuide.extraLightBlue(),
+                    .underlineStyle: NSNumber(value: 2),
+                    .underlineColor: WPStyleGuide.wordPressBlue()]
+        case .del:
+            return [.backgroundColor: WPStyleGuide.extraLightRed(),
+                    .underlineStyle: NSNumber(value: 2),
+                    .underlineColor: WPStyleGuide.errorRed(),
+                    .strikethroughStyle: NSNumber(value: 1),
+                    .strikethroughColor: UIColor.black]
+        default:
+            return nil
+        }
+    }
+}
+
+// This should be moved to WPShared on Revisions V2
+//
+private extension WPStyleGuide {
```

Comment:
There are a few extensions for `WPStyleGuide` in `WPiOS` already. If we put this in `WPStyleGuide+ DiffAbstractValue.swift` it could stay as-is.

Reply:
Great! Thanks for the hint! I'll create an extension for that!